typescript/packages/plugins/superfluid/src/tools.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    flow,
+    getFlowrate,
+    updateMemberUnits,
+    getUnits,
+    getMemberFlowRate,
+    getTotalFlowRate,
+} from "./methods";
+import {
+    flowParametersSchema,
+    getFlowrateParametersSchema,
+    updateMemberUnitsParametersSchema,
+    getUnitsParametersSchema,
+    getMemberFlowRateParametersSchema,
+    getTotalFlowRateParametersSchema,
+} from "./parameters";
+import type { DeferredTool, EVMWalletClient } from "@goat-sdk/core";
+import type { z } from "zod";
+
+export function getTools(): DeferredTool<EVMWalletClient>[] {
```

Comment:
We actually updated the core. You'll need to rebase and see how the other plugins are implemented. No need for DeferredTool anymore!